src/source/ExplicitEulerSource.cpp

```diff
@@ -5,20 +5,17 @@
 /// \copyright  <2015-2020> Forschungszentrum Juelich GmbH. All rights reserved.
 
 #include "ExplicitEulerSource.h"
-#include "../utility/Parameters.h"
 #include "../boundary/BoundaryController.h"
 
-ExplicitEulerSource::ExplicitEulerSource() {
-    auto params = Parameters::getInstance();
-
-    m_dt = params->get_real("physical_parameters/dt");
-    m_dir_vel = params->get("solver/source/dir");
+ExplicitEulerSource::ExplicitEulerSource(Settings const &settings) {
+    m_dt = settings.get_real("physical_parameters/dt");
```

Comment:
remove member variable `m_dt` instead access value through Settings

Reply:
done in 0d1d248